### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by marvinpfoertner**
# In a Nutshell
This PR adds a proposal for a CODEOWNERS file.

### CODE REVIEW

**Comment by marvinpfoertner** [comment]
@JonathanWenger I propose that we merge this, unless there is any obvious problems with the current assignments. Then we can discuss the assignment in person in the next software meeting.

**Comment by JonathanWenger** [comment]
Before we require reviews from code owners i propose we discuss whether secondary code owners are sensible for redundancy and fast turnarounds.

**Review by JonathanWenger — APPROVED** [review]

**Pull request merged**
